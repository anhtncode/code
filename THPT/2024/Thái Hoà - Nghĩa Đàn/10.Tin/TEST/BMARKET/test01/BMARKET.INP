13 37 112