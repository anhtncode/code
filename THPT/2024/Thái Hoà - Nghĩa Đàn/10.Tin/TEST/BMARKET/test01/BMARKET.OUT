24 61 98 